Dockerfile

```diff
@@ -112,20 +95,26 @@ RUN docker-php-ext-install bcmath xsl
 
 RUN apt-get install -y mariadb-client
 
-RUN composer global require hirak/prestissimo
+# Install Drupal.
+WORKDIR /var/www
+
+RUN rm -rf html
+RUN composer create-project drupal/legacy-project:^8.9 html
```

Comment:
This is the biggest change. What I was finding was that there were implicit assumptions in `Robofile.php` about the prescence of `wikimedia/composer-merge-plugin`. In 8.7, core required that out of the boc, which then also meant that core's `require-dev` dependencies came along for the ride and were installed as well. In 8.9, that's no longer the case. `wikimedia/composer-merge-plugin` is no longer used, and the core dev dependencies were not included. Given that, I decided to remove the use of the `duc` utility and use `composer create-project`. Here we start from `drupal/legacy-project` to give us a simplified file layout, the same as what we had. We then layer on `wikimedia/composer-merge-plugin` (we still want that for the purposes of merging the module under test's require and require-dev dependencies) and `drupal/core-dev` to get what we need to run tests.

Reply:
TIL's about legacy-project, nice find! I imagine we'll have to change that later (given "legacy"), but that could always be when running against Drupal 9.